package.json

```
@@ -18,7 +18,7 @@
   },
   "productName": "vite-reactts-electron-starter",
   "scripts": {
-    "dev": "concurrently \"yarn dev:vite\" \" yarn dev:electron\"",
+    "dev": "concurrently \"npm run dev:vite\" \" npm run dev:electron\"",
     "dev:vite": "vite",
```

This seems missing too:
```suggestion
    "dev:vite": "npm run build:vite && vite",
```